### DISCUSSION TITLE
Errors reading Argo profile data into METplus

### DISCUSSION THREAD

**Discussion by GwenChen-NOAA** [dtcenter/METplus#1968] [File I/O] [answered]
Dear METplus team,

EMC EVS project requests to use Argo profile data to verify RTOFS ocean temperature and salinity forecasts.  I tried to read the data files available on WCOSS2 at /lfs/h1/ops/dev/dcom/$YYYYMMDD/validation_data/marine/argo using PointStat.  They are netCDF files and temperature data is listed as TEMP(N_PROF, N_LEVELS).  I set OBS_VAR1_NAME = TEMP and OBS_VAR1_LEVELS = "(*,0)" in PointStat to test run sea level verification, but I got the following error message:

ERROR  : VarInfo::set_level_info_grib() - failed to parse level string '(*,0)'

Can PointStat automatic recognize netCDF files?  Or I need to specify it in PointStat config files?  Also, Argo profile data have more than 2000 levels.  Does it cause a problem for PointStat?  Please let me know if you don't have access to WCOSS2 and need me to upload a sample data file.  Thank you very much for your help and have a great holiday!

**Comment by JohnHalleyGotway** [comment, marked as answer]
Hi @GwenChen-NOAA, I see you'd like to make use of the Argo profile data to verify forecasts of ocean temperature and salinity. As you've already found, no, MET does not support reading this point observation data in its native format.

However, using the python embedding of point observations would likely be a great option. I'd recommend that we work together to setup a python script to serve up this point observation data to the MET tools, and then ideally add a METplus use case to demonstrate that usage.

**Comment by GwenChen-NOAA** [comment]
John,

Sorry for the late response.  EVS Alpha Test deadline was last Thursday
(12/1), and we were all focused on meeting the deadline.  Alicia asked me
to create a new METplus issue for a new use case, and I just did.  You may
close this discussion and we will follow up with METplus issue #1977
<https://github.com/dtcenter/METplus/issues/1977>.  Thank you!

Gwen
